oracle/cpi.yml

```diff
@@ -0,0 +1,109 @@
+---
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-oracle-cpi
+    url: file://((local_oracle_cpi_release)) 
+
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: file://((local_oracle_light_stemcell))
+
+# Configure instance shapes 
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    availability_domain: ((ad))
+    instance_shape: VM.Standard1.2
+
+# Configure VCN and subnet
+- type: replace
+  path: /networks/name=default/subnets/0/cloud_properties?
+  value:
+    vcn: ((vcn))
+    subnet_name: ((subnet))
+
+# Enable registry job
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: registry
+    release: bosh
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/registry?
+  value:
+    address: ((internal_ip))
+    host: ((internal_ip))
+    db: # todo remove
+      host: 127.0.0.1
+      user: postgres
+      password: ((postgres_password))
+      database: bosh
+      adapter: postgres
+    http:
+      user: registry
+      password: ((registry_password))
+      port: 25777
+    username: registry
+    password: ((registry_password))
+    port: 25777
+
+# Oracle Cloud Infrastructure properties for BOSH
+- type: replace
+  path: /instance_groups/name=bosh/properties/oci?
+  value: &cpi_conf
+    user: ((user))
+    fingerprint: ((fingerprint))
+    tenancy: ((tenancy))
+    region: ((region))
+    compartment: ((compartment))
+    apikey: ((apikey))
+    cpikey: ((cpikeys.private_key))
+    cpiuser: ((provisioned_username))
+    authorized_keys:
+       user: ((ssh_key))
```

Comment:
Is this required? In general, we avoid recommending users provisioning hard-coded keys as part of the base VM image (in general, all access should be through `bosh ssh` or release-specific access controls). If we were able to avoid this, it's one less thing users need to be configuring in the vars, and optimally it works with it.

Reply:
No, it's optional.  This was added when our "older" custom built stemcell didn't support `bosh ssh`.  I'll look into removing it from cpi.yml.